src/libopenrave-core/environment-core.h

```diff
@@ -830,12 +846,7 @@ class Environment : public EnvironmentBase
         }
         }
 
-        if( listbodies.size() == 1 ) {
```

Comment:
Why is this check removed?

Reply:
Liboon made the change to get rid of a single body condition. I think if you follow the code and get to `JsonWriter::Write` it is the same.

https://github.com/rdiankov/openrave/blob/b6c8feaacd4216f3de180c755d4acfb10b0b50b1/src/libopenrave-core/jsonparser/jsonwriter.cpp#L57-L65